sway-lib-std/src/u256.sw

```diff
@@ -638,3 +678,77 @@ impl core::ops::Divide for U256 {
         quotient
     }
 }
+
+impl Power for U256 {
+    /// Raises self to the power of `exp`, using exponentiation by squaring.
+	///
+	/// # Panics
+	///
+	/// Panics if the result overflows the type.
+    fn pow(self, expon: u32) -> Self {
```

Comment:
It says `exp` in the doc comment, and it's `expon` here, and it's `exponent` in the definition of the `Power` trait :smile: 

I would go for `exponent` to have it consistent with other `Power` impls.

Reply:
Done.